### PR TITLE
Adding Chartbeat tests for Canonical and AMP MAPs

### LINKED ISSUE #5788: Add Chartbeat tests for MAPs & Live Radio
Front pages and articles have E2E tests enabled for Chartbeat. While we are happy with Chartbeat functioning as expected, it would be good to ensure we're aware of when it is missing from a page as this will lead to incorrect reporting for editorial teams.

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Resolves #5788

**Overall change:** 
Adding tests for Chartbeat on Canonical and AMP MAPs

**Code changes:**
Added tests for Chartbeat

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by karinathomasbbc — APPROVED** [review]

File: cypress/integration/pages/mediaAssetPage/testsForAMPOnly.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
...
     });
+
+    describe('Chartbeat', () => {
+      if (envConfig.chartbeatEnabled) {
+        it('should have chartbeat config UID', () => {
+          cy.hasAmpChartbeatConfigUid(0);
+        });
+      }
+    });
     describe('AMP Status', () => {
```
Comment (karinathomasbbc) [inline cypress/integration/pages/mediaAssetPage/testsForAMPOnly.js:46]: Can we add a new line here plz?

Comment (rebeccamcginn) [reply cypress/integration/pages/mediaAssetPage/testsForAMPOnly.js:46]: Done!

File: cypress/support/commands/analytics.js [hunk 2]
```diff
@@ -29,8 +29,8 @@ Cypress.Commands.add('hasGlobalChartbeatConfig', () => {
 });
 
 // Should be moved into integration/pages/index.js once all pages have Chartbeat
-Cypress.Commands.add('hasAmpChartbeatConfigUid', () => {
+Cypress.Commands.add('hasAmpChartbeatConfigUid', (position = 1) => {
   cy.get('amp-analytics script[type="application/json"]')
-    .eq(1)
+    .eq(position)
```
Comment (ryanmccombe) [inline cypress/support/commands/analytics.js]: Do we need this`eq` at all?  @karinathomasbbc had a similar issue with script tags being in different positions - it seemed to work fine with the `.eq` removed, and made it such that it doesn't matter what order the script tags are in: https://github.com/bbc/simorgh/commit/f399feb3cb1d05ed545b4fe110763202e990e770

Comment (rebeccamcginn) [reply cypress/support/commands/analytics.js]: Removed :)

Comment (karinathomasbbc) [reply cypress/support/commands/analytics.js]: 👍

**Review by ryanmccombe — APPROVED** [review]

**Pull request merged**